devnet-sdk/devstack/example/example_test.go

```diff
@@ -19,6 +23,23 @@ func TestExample1(gt *testing.T) {
 func TestExample2(gt *testing.T) {
 	t := devtest.ParallelT(gt)
 	sys := SimpleInterop(t)
+	require := t.Require()
+
+	alice := sys.Wallet.NewUser(sys.L2ELA)
+	sys.FaucetA.Fund(alice.Address(), eth.GWei(100))
+
+	bob := sys.Wallet.NewUser(sys.L2ELA)
+	require.Zero(bob.Balance())
+
+	pre := alice.Balance()
+	transferred := eth.GWei(42)
+	tx := alice.Transfer(bob.Address(), transferred)
+	require.Equal(params.TxGas, tx.Included.Value().GasUsed, "transfers cost 21k gas")
+
+	postAlice := alice.Balance()
+	postBob := bob.Balance()
+	require.True(postAlice.Add(transferred).Lt(pre), "alice must pay a fee")
+	require.Equal(transferred, postBob, "bob must receive the exact amount")
```

Comment:
To avoid the getter, I'd make this:
```
alice.VerifyBalanceLessThan(eth.Gwei(100 - 42), "alice must pay a fee")
bob.VerifyBalance(transferred, "bob must receive the exact amount")
```

We could have a `VerifyBalance` that takes a precondition but there's a pretty limited number of comparisons so I think I'd just have separate methods in this case to keep the test clean (following the principle of pushing complexity into the DSL).

Reply:
done